### PR TITLE
Configure localdirect.gov.uk as a global archive

### PR DESCRIPTION
**Pull request by issyl0**

### CODE REVIEW

File: data/transition-sites/dclg_localdirect.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_localdirect
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
```
Comment (jennyd) [inline data/transition-sites/dclg_localdirect.yml]: The site [does have some TNA crawls](http://webarchive.nationalarchives.gov.uk/*/http://www.localdirect.gov.uk), so we can add a real timestamp here.

According to the [ticket](https://govuk.zendesk.com/agent/tickets/1249417) a final TNA crawl is under way and we'll need to update the timestamp again before the site transitions, so could you add a comment to that effect here?

File: data/transition-sites/dclg_localdirect.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_localdirect
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
```
Comment (jennyd) [inline data/transition-sites/dclg_localdirect.yml]: There's a [short url for DCLG](https://github.com/alphagov/transition-config/blob/master/data/transition-sites/dclg.yml#L6) which you could add as `homepage_furl`.

File: data/transition-sites/dclg_localdirect.yml (hunk 1 above)
Comment (issyl0) [inline data/transition-sites/dclg_localdirect.yml]: Sorry. I thought "TNA crawl underway" meant "none yet" - should have checked! :disappointed: Anyway, done. :relieved:

File: data/transition-sites/dclg_localdirect.yml (hunk 2 above)
Comment (jennyd) [inline data/transition-sites/dclg_localdirect.yml]: ...and there's even [`css: department-for-communities-and-local-government`](https://github.com/alphagov/transition-config/blob/76ffb59fc56d6be64d83a0b4283648c76be3b5d1/data/transition-sites/dclg.yml#L12) too!

Comment (issyl0) [inline data/transition-sites/dclg_localdirect.yml]: Added short URL and CSS - thanks.

**Comment by jennyd** [comment]
:+1:

**Pull request merged**